extensions/blocks/rating-star/index.js

```diff
@@ -15,7 +15,7 @@ import './style.scss';
 export const name = 'rating-star';
 
 export const settings = {
-	title: 'Star Rating',
+	title: ( 'Star Rating', 'jetpack' ),
```

Comment:
```suggestion
	title: __( 'Star Rating', 'jetpack' ),
```

Reply:
Ugh, not sure how that happened, thank you!